macos/Runner/DebugProfile.entitlements

```diff
@@ -8,5 +8,9 @@
 	<true/>
 	<key>com.apple.security.network.server</key>
 	<true/>
+	<key>com.apple.security.network.client</key>
+    <true/>
```

Comment:
This probably wants to be indented?

Reply:
it works on my machine 
<img width="581" alt="image" src="https://github.com/AerospaceNU/flutter_groundstation/assets/88739911/1cba11d1-e490-4630-b3c0-ea39dae3162d">

Reply:
fixed it :)